configserver/src/main/java/com/yahoo/vespa/config/server/deploy/ModelContextImpl.java

```diff
@@ -179,6 +180,7 @@ public FeatureFlags(FlagSource source, ApplicationId appId) {
             this.useBucketExecutorForLidSpaceCompact = flagValue(source, appId, Flags.USE_BUCKET_EXECUTOR_FOR_LID_SPACE_COMPACT);
             this.enableFeedBlockInDistributor = flagValue(source, appId, Flags.ENABLE_FEED_BLOCK_IN_DISTRIBUTOR);
             this.maxDeadBytesRatio = flagValue(source, appId, Flags.MAX_DEAD_BYTES_RATIO);
+            this.clusterControllerMaxHeapSizeInMb = flagValue(source, appId, Flags.CLUSTER_CONTROLLER_MAX_HEAP_SIZE_IN_MB);
```

Comment:
Yeah, sorry, missed this.

Reply:
But that enum constant I don't see!

Reply:
Added in an earlier PR (https://github.com/vespa-engine/vespa/pull/16463), that @baldersheim merged before I added any reviewers, so forgot to add you afterwards.